mocks/mocks.py

```diff
@@ -15,12 +15,39 @@
 
 import ccdproc
 
+import pynbody
+
 import gunagala
 from gunagala.utils import ensure_unit
 
 from mocks.utils import load_yaml_config
 
 
+def read_gadget(data_path):
+    """
+    Reads the raw output of the Genesis simulations
+
+    Parameters
+    ----------
+    data_path : str
+        The path to the data.
+
+    Returns
+    -------
+    particle_pos: pynbody.array.SimArray
+        The positions of the simulation particles (only stars).
+    particle_value: pynbody.array.SimArray
+        The mass of the simulation particles (only stars).
+    sim_properties: pynbody.simdict.SimDict
+        Contains cosmological information of the simulation.
+    """
+    sim_data = pynbody.load(data_path)
+    particle_pos = sim_data.stars['pos'].in_units('Mpc')
+    particle_value = sim_data.stars['mass'].in_units('Msol')
+    sim_properties = sim_data.properties
+    return particle_pos, particle_value, sim_properties
```

Comment:
Really not sure if this function is necessary if it only ends up breaking up the pybody structure. Better to keep things in tact... but maybe the problem is that I don't know your workflow - I need to see the jupyter notebook to understand how this function fits into the flow.

Reply:
Let's close this PR for now and open a new one soon that provides that flow.